docker-compose.yml

```diff
@@ -52,6 +52,11 @@ services:
     volumes:
       - ./kbs/data/reference-values:/opt/confidential-containers/attestation-service/reference_values:rw
       - ./kbs/config/rvps.json:/etc/rvps.json:rw
+    command: [
```

Comment:
Not sure other verifier, I think `SE` does have something related document here https://github.com/confidential-containers/trustee/blob/main/deps/verifier/src/se/README.md need be revised accordingly, maybe need a new PR to address this change in SE doc. CC @stevenhorsman @bpradipt

Reply:
Thanks to the notice - cc @gauravkuredhat

Reply:
oh yeah I didn't update any of the docs. I will add a commit to do that

Reply:
updated